src/io/flutter/perf/FlutterWidgetPerfManager.java

```diff
@@ -318,6 +324,21 @@ public void dispose() {
     if (currentStats != null) {
       currentStats.dispose();
       currentStats = null;
+      listeners.clear();
```

Comment:
Should we instead call `removePerfListener` for each listener?

Reply:
I don't think we need  to as the stats object the listeners are added to is disposed a couple lines earlier.